### DISCUSSION TITLE
Updating 'Razor Kebbit Tracking' Script to make the object  clearer

### DISCUSSION THREAD

**Discussion by AC1888** [runelite/runelite#18473] [Ideas]
Hi,

I'm looking to get @cwjoshuak 'Razor Kebbit Tracking' script updated to work similarly to the 'Object Marker' Script?

The object marker script has options to 'Fill' the 'Hull' of the object, meaning you can have a solid color filling the object completely meaning its a lot easier to click the specified object and see it more clearly.

the razor kebbit script has something similar where it highlights the object but doesnt allow you to fill it in completely. 

ive attached a few screenshots the show this. the object marker has completely filled burrow in red where as the razor kebbit script only highlights the bush in orange and isn't anywhere near as clear as the object marker.

 
![razor Kebbit Tracking](https://github.com/user-attachments/assets/a03ca6b2-1ce4-448d-a45f-272966790544)
![Object Marker script](https://github.com/user-attachments/assets/d9b6095e-3fff-4edb-acc3-6ffb7e5fee96)
![screenshot of hunter](https://github.com/user-attachments/assets/85541aad-9a63-455e-adb8-051ee60e6057)

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.
